src/test/java/com/github/awwkoala/selenium/CheckBoxesTests.java

```diff
@@ -0,0 +1,36 @@
+package com.github.awwkoala.selenium;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class CheckBoxesTests {
+/*  Stwórz test automatyczny na podstawie poniższego scenariusza testowego:
+  Przejdź na stronę checkboxes
+  Sprawdź, że po wejściu na stronę checkbox 1 jest odznaczony, zaś checkbox 2 jest zaznaczony
+  Zaznacz checkbox 1, odznacz checkbox 2
+  Sprawdź, że checkbox 1 jest zaznaczony, zaś checkbox 2 jest odznaczony*/
+
+  @Test
+  public void checkBoxesTest() {
+    System.setProperty("webdriver.chrome.driver", "/opt/WebDriver/bin/chromedriver");
```

Comment:
This isn't good code for test body. It's a part of WebDriver initialization process. So I would create such utility called `WebDriverProvider` or sth like that which have a responsibility to prepare, initialize and provide chrome driver in the TestBase